controllers/datamovement_controller.go

```diff
@@ -456,6 +466,26 @@ func buildDMCommand(profile dmConfigProfile, hosts []string, dm *nnfv1alpha1.Nnf
 	cmd = strings.ReplaceAll(cmd, "$SRC", dm.Spec.Source.Path)
 	cmd = strings.ReplaceAll(cmd, "$DEST", dm.Spec.Destination.Path)
 
+	// Allow the user to override settings
+	if userConfig {
+
+		// Add extra DCP options from the user
+		if len(dm.Spec.UserConfig.DCPOptions) > 0 {
+			opts := dm.Spec.UserConfig.DCPOptions
+
+			// Insert the extra dcp options before the src argument
+			if strings.Contains(cmd, "dcp") {
+				idx := strings.Index(cmd, dm.Spec.Source.Path)
+				if idx != -1 {
+					cmd = cmd[:idx] + opts + " " + cmd[idx:]
```

Comment:
If it found the dcp command but not the source path, is that a reason to give an error?

Reply:
Good call. I've added an error message.